src/model.rs

```diff
@@ -159,10 +160,21 @@ pub enum Status {
 
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, FromJsonQueryResult)]
 pub struct StatusList {
-    pub bits: usize,
+    pub bits: u8,
     pub lst: String,
 }
 
+/// Request payload for publishing and updating a status list token
+#[derive(Deserialize, Serialize, Clone)]
+pub struct StatusListTokenPayload {
```

Comment:
Could you use a more suitable name? That could be used for both publishing and updating.
```suggestion
pub struct StatusRequest {
```